### ISSUE TITLE
Proposal: have guide examples open a new file rather that overwrite the existing file

### ISSUE THREAD

**Issue by bbarker** [google/codeworld#1501]
I've seen my child make this error a few times, whereby he'll click an example in the guide and it will overwrite what is in the currently opened CodeWorld file. Today he somehow ended up permanently (as far as I can tell) deleting the work he had in an existing file. It seems like it may be more desirable to either create a new file when the user interacts with an example in the guide, or append what is in the example to the currently open file.

**Comment by cdsmith** [comment]
Good point!  I really like this, and it should be easy to do.  Even aside from overwriting the existing code, it would be valuable to be able to try out an example without closing the Guide and making students find their way back to it.
